cycles.json

```diff
@@ -140,5 +140,12 @@
         "packs": [
             "DotE"
         ]
+    },
+    {
+        "id": "premium",
+        "name": "Premium Packs",
```

Comment:
I'm not sure how we want to organize future, non-cycle releases.

Reply:
seems sensible to me to group them in this way. i concur.